sn_networking/src/lib.rs

```diff
@@ -475,6 +475,12 @@ impl Network {
             .map_err(|_e| Error::InternalMsgChannelDropped)
     }
 
+    /// Notify the node got paid once
+    pub fn notify_got_paid(&self) -> Result<()> {
```

Comment:
lets change this to `notify_payment_received` and `GotPaid` above to `PaymentReceived`